File: hackerrank/javascript/week1/plusminus.js
function plusMinus(arr) {
    // Write your code here
    let positiveNums = 0
    let negativeNums = 0
    let zeroes = 0
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] < 0) {
            negativeNums += 1
        }
         if (arr[i] > 0) {
            positiveNums += 1
        }
         if (arr[i] === 0) {
            zeroes += 1
        }
    }
    console.log((positiveNums/arr.length).toFixed(6))
    console.log((negativeNums/arr.length).toFixed(6))
    console.log((zeroes/arr.length).toFixed(6))
}

File: hackerrank/python/week1/plusminus.py
def plusMinus(arr):
    # Write your code here
    length = len(arr)
    positiveCount = 0
    zeroes = 0
    negativeCount = 0
    for i in range(length):
        if arr[i] > 0:
            positiveCount += 1
        if arr[i] == 0:
            zeroes += 1
        if arr[i] < 0:
            negativeCount += 1
    print(format(positiveCount/length, '.6f'))
    print(format(negativeCount/length, '.6f'))
    print(format(zeroes/length, '.6f'))
